src/main/kotlin/com/terraformation/backend/seedbank/db/AccessionStore.kt

```diff
@@ -936,13 +936,13 @@ class AccessionStore(
             .and(ACCESSIONS.STATE_ID.`in`(AccessionState.activeValues)))
   }
 
-  fun getSummaryStatistics(projectId: ProjectId): AccessionSummaryStatistics {
+  fun getSummaryStatistics(facilityId: FacilityId, projectId: ProjectId): AccessionSummaryStatistics {
```

Comment:
Since this is currently only used in one place, I figured it was OK to modify it directly instead of making another function that takes both `facilityId` and `projectId`